src/OnlineRetailPortal.Services/Services/ProductService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using OnlineRetailPortal.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using OnlineRetailPortal.Mock;
+
+namespace OnlineRetailPortal.Services
+{
+    public class ProductService : IProductService
+    {
+        IProductStoreFactory productStoreFactory = new Mock.ProductStoreFactory();
+        IProductStore productStore ;
+        Core.Product product;
```

Comment:
why do we need this at root level? also the dependencies should be injected in a constructor

Reply:
added only to test